napari/layers/points/points.py

```diff
@@ -262,16 +262,21 @@ def __init__(
         blending='translucent',
         visible=True,
     ):
+        if ndim is None and scale is not None:
+            ndim = len(scale)
         if data is None:
             if ndim is None:
                 ndim = 2
             data = np.empty((0, ndim))
         else:
             data = np.atleast_2d(data)
             data_ndim = data.shape[1]
-            if ndim is not None and ndim != data_ndim:
-                raise ValueError("Points dimensions must be equal to ndim")
-            ndim = data_ndim
+            if data_ndim != 0:
+                if ndim is not None and ndim != data_ndim:
+                    raise ValueError("Points dimensions must be equal to ndim")
+                ndim = data_ndim
+        if ndim is None:
```

Comment:
Same above here.

Reply:
@goanpeca does black not specify line spacing for us? Or are single spaces in functions preserved?

Reply:
I am not sure if black handles that but the lack of spaces there seems weird.